draft-ietf-ppm-dap.md

```diff
@@ -1498,30 +1531,57 @@ If the upload request is malformed, the Leader aborts with error
 If the Leader does not recognize the task ID, then it aborts with error
 `unrecognizedTask`.
 
+Otherwise, the Leader responds with a body consisting of an `UploadResponse` and
+the media type `application/dap-upload-resp`. The structure of the response
+is shown below:
+
+~~~ tls-presentation
+struct {
+  ReportID id;
+  ReportError error;
+} ReportUploadStatus;
+
+struct {
+  ReportUploadStatus status[status_count];
```

Comment:
As above, `status_count` is misleading because the value here is a number of bytes. Let's use `content_length`. And we should make it it clear that the response's `Content-Length` header MUST match the value used here.